### DISCUSSION TITLE
Mitsubishi MIEL HVAC console spam after 13.0

### DISCUSSION THREAD

**Discussion by host-felice** [arendst/Tasmota#19189] [Q&A] [answered]
### PROBLEM DESCRIPTION
After upgrading to 13.0 the console gets really flooded with "power on" mqtt messages. Reverting back to 12.4 solves the problem.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): wemos d1 mini
- [ ] Tasmota binary firmware version number used: 13.0
  - [ ] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: OTA update
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
9:57:50.091 CMD: Backlog Template; Module; GPIO 255
09:57:50.162 MQT: stat/clima4/RESULT = {"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":0,"BASE":18}
09:57:50.364 MQT: stat/clima4/RESULT = {"Module":{"18":"Generic"}}
09:57:50.573 MQT: stat/clima4/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"5376":"MiEl HVAC Tx"},"GPIO2":{"0":"None"},"GPIO3":{"5408":"MiEl HVAC Rx"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"0":"None"},"GPIO13":{"0":"None"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
09:58:44.557 CMD: status 0
09:58:44.568 MQT: stat/clima4/STATUS = {"Status":{"Module":18,"DeviceName":"Clima4","FriendlyName":["Clima4"],"Topic":"clima4","ButtonTopic":"0","Power":1,"PowerOnState":0,"LedState":0,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":1,"InfoRetain":0,"StateRetain":1,"StatusRetain":0}}
09:58:44.574 MQT: stat/clima4/STATUS1 = {"StatusPRM":{"Baudrate":2400,"SerialConfig":"8E1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-minimal.bin.gz","RestartReason":"Software/System restart","Uptime":"0T00:07:03","StartupUTC":"2023-07-25T07:51:41","Sleep":50,"CfgHolder":4617,"BootCount":14,"BCResetTime":"2022-09-09T21:53:23","SaveCount":318,"SaveAddress":"F9000"}}
09:58:44.578 MQT: stat/clima4/STATUS2 = {"StatusFWR":{"Version":"13.0.0(TasmoCompiler-esp8266generic)","BuildDateTime":"2023-07-24T18:39:56","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"362/699"}}
09:58:44.582 MQT: stat/clima4/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["VillaTorreZisa",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008029","2805C80001000600003C5A0A192800000000","00000080","00006000","10004080","00000000"]}}
09:58:44.593 MQT: stat/clima4/STATUS4 = {"StatusMEM":{"ProgramSize":544,"Free":456,"Heap":25,"ProgramFlashSize":1024,"FlashSize":4096,"FlashChipId":"164068","FlashFrequency":40,"FlashMode":"DOUT","Features":["00000809","0F804582","04260001","00000003","00000000","C000E881","04000004","00001000","54000000","00000080"],"Drivers":"1,2,4,6,8,9,10,12,18,19,22,24,26,29,30,35,37,44,45,62,68","Sensors":"1,2"}}
09:58:44.601 MQT: stat/clima4/STATUS5 = {"StatusNET":{"Hostname":"wm-clima4-94","IPAddress":"192.168.178.94","Gateway":"192.168.178.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.178.1","DNSServer2":"0.0.0.0","Mac":"E8:9F:6D:91:FE:1E","Webserver":2,"HTTP_API":1,"WifiConfig":5,"WifiPower":17.0}}
09:58:44.605 MQT: stat/clima4/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.178.26","MqttPort":1883,"MqttClientMask":"clima4","MqttClient":"clima4","MqttUser":"Felice","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
09:58:44.611 MQT: stat/clima4/STATUS7 = {"StatusTIM":{"UTC":"2023-07-25T07:58:44","Local":"2023-07-25T09:58:44","StartDST":"2023-03-26T02:00:00","EndDST":"2023-10-29T03:00:00","Timezone":99,"Sunrise":"05:49","Sunset":"20:32"}}
09:58:44.617 MQT: stat/clima4/STATUS10 = {"StatusSNS":{"Time":"2023-07-25T09:58:44","MiElHVAC":{"Temperature":30.5,"Operation":"OFF","Compressor":"OFF","roomtemp":"030000140001BDBDFE42000078780000","status":"060000000000100EFB00004200000000","stage":"09000000000000000000000000000000"},"TempUnit":"C"}}
09:58:44.623 MQT: stat/clima4/STATUS11 = {"StatusSTS":{"Time":"2023-07-25T09:58:44","Uptime":"0T00:07:03","UptimeSec":423,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":21,"MqttCount":1,"POWER":"ON","Wifi":{"AP":1,"SSId":"VillaTorreZisa","BSSId":"E8:DF:70:4C:2A:5B","Channel":1,"Mode":"11n","RSSI":64,"Signal":-68,"LinkCount":1,"Downtime":"0T00:00:03"}}}

```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
10:00:00.132 SRC: Switch
10:00:00.138 MQT: stat/clima4/RESULT = {"POWER":"ON"}
10:00:00.142 MQT: stat/clima4/POWER = ON (retained)
10:00:00.181 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
10:00:00.383 MiElHVAC: response data 030000140001BDBCFE42000078780000
10:00:00.433 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
10:00:00.636 MiElHVAC: response data 0200000103050207000083B446000000
10:00:00.638 SRC: Switch
10:00:00.642 MQT: stat/clima4/RESULT = {"POWER":"ON"}
10:00:00.648 MQT: stat/clima4/POWER = ON (retained)
10:00:00.691 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74

```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Just turn on the AC and you will see the problem

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
to have only 1 on confirmation as in 12.4 versione

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
<img width="587" alt="2023-07-25" src="https://github.com/arendst/Tasmota/assets/36647269/c3811fca-351f-42f9-a125-52b5747c72be">

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
5 out of 5 AC show the same problem. They work, but the MQTT server gets flooded.

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
You have an other issue. mqtt connection or reliability. You have always `retained` in your mqtt messages. This happens when mqtt connection gets lost.

**Reply by host-felice** [reply]
No Jason, it happens on 13.0. Connection is very good in all my wemos and the problem disappears on 12.4 and previous versions. 

Something is wrong with 13.0. Could you please move it back to issues?

Thank you.

**Reply by Jason2866** [reply]
No, as long there are retained mqtt messages. Solve the retain issue.

**Reply by host-felice** [reply]
There is no retain problem! PowerRetain on is not a problem, it is an option. I cleaned the mqtt broker and only tasmota sends the retain flag. 13.0 flood, 12.4 (and previous versions) no flood. 

Again, could you please move back my post to issues?

**Reply by arendst** [reply]
Enable more logging and witness the cause of the power issue.

**Comment by host-felice** [comment]
> Enable more logging and witness the cause of the power issue.

weblog 4. I REMOVED everything under stat/clima4 in the mqtt broker. PowerRetain off. The wifi signal is 70%. This is the result:
13:57:02.137 SRC: Switch
13:57:02.142 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:02.146 MQT: stat/clima4/POWER = ON
13:57:02.187 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:57:02.389 MiElHVAC: response data 09000000020000000000000000000000
13:57:02.439 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:02.540 WIF: Sending Gratuitous ARP
13:57:02.642 MiElHVAC: response data 0200000103050207000083B446000000
13:57:02.644 SRC: Switch
13:57:02.650 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:02.653 MQT: stat/clima4/POWER = ON
13:57:02.697 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:57:02.899 MiElHVAC: response data 060000000100130EFC00004200000000
13:57:02.906 MQT: tele/clima4/SENSOR = {"Time":"2023-07-25T13:57:02","MiElHVAC":{"Temperature":30.0,"Operation":"ON","Compressor":"OFF","roomtemp":"030000140001BCBCFE420000787D0000","status":"060000000100130EFC00004200000000","stage":"09000000020000000000000000000000"},"TempUnit":"C"}
13:57:02.910 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:03.112 MiElHVAC: response data 0200000103050207000083B446000000
13:57:03.115 SRC: Switch
13:57:03.119 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:03.123 MQT: stat/clima4/POWER = ON
13:57:03.163 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:57:03.366 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:57:03.416 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:03.618 MiElHVAC: response data 0200000103050207000083B446000000
13:57:03.620 SRC: Switch
13:57:03.625 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:03.630 MQT: stat/clima4/POWER = ON
13:57:03.674 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:57:03.876 MiElHVAC: response data 09000000020000000000000000000000
13:57:03.927 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:04.130 MiElHVAC: response data 0200000103050207000083B446000000
13:57:04.132 SRC: Switch
13:57:04.137 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:04.141 MQT: stat/clima4/POWER = ON
13:57:04.181 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:57:04.383 MiElHVAC: response data 060000000100130EFC00004200000000
13:57:04.435 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:04.637 MiElHVAC: response data 0200000103050207000083B446000000
13:57:04.639 SRC: Switch
13:57:04.644 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:04.647 MQT: stat/clima4/POWER = ON
13:57:04.693 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:57:04.895 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:57:04.945 WIF: Checking connection...
13:57:04.948 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:05.150 MiElHVAC: response data 0200000103050207000083B446000000
13:57:05.152 SRC: Switch
13:57:05.157 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:05.161 MQT: stat/clima4/POWER = ON
13:57:05.165 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:57:05.367 MiElHVAC: response data 09000000020000000000000000000000
13:57:05.418 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:05.620 MiElHVAC: response data 0200000103050207000083B446000000
13:57:05.622 SRC: Switch
13:57:05.627 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:05.631 MQT: stat/clima4/POWER = ON
13:57:05.675 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:57:05.904 MiElHVAC: response data 060000000100130EFC00004200000000
13:57:05.909 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:06.111 MiElHVAC: response data 0200000103050207000083B446000000
13:57:06.113 SRC: Switch
13:57:06.118 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:06.122 MQT: stat/clima4/POWER = ON
13:57:06.163 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:57:06.366 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:57:06.417 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:06.619 MiElHVAC: response data 0200000103050207000083B446000000
13:57:06.622 SRC: Switch
13:57:06.626 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:06.631 MQT: stat/clima4/POWER = ON
13:57:06.675 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:57:06.877 MiElHVAC: response data 09000000020000000000000000000000
13:57:06.927 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:07.130 MiElHVAC: response data 0200000103050207000083B446000000
13:57:07.132 SRC: Switch
13:57:07.136 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:07.141 MQT: stat/clima4/POWER = ON
13:57:07.182 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:57:07.385 MiElHVAC: response data 060000000100130EFC00004200000000
13:57:07.436 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:07.638 MiElHVAC: response data 0200000103050207000083B446000000
13:57:07.640 SRC: Switch
13:57:07.645 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:07.649 MQT: stat/clima4/POWER = ON
13:57:07.695 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:57:07.897 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:57:07.948 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:08.151 MiElHVAC: response data 0200000103050207000083B446000000
13:57:08.153 SRC: Switch
13:57:08.157 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:08.162 MQT: stat/clima4/POWER = ON
13:57:08.165 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:57:08.368 MiElHVAC: response data 09000000020000000000000000000000
13:57:08.420 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:57:08.622 MiElHVAC: response data 0200000103050207000083B446000000
13:57:08.624 SRC: Switch
13:57:08.629 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:08.632 MQT: stat/clima4/POWER = ON


and so on.

this is after I turn off the AC:

13:58:28.380 MiElHVAC: response data 0200000003050207000083B446000000
13:58:28.430 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:28.633 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:28.689 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:28.890 MiElHVAC: response data 0200000003050207000083B446000000
13:58:28.941 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:29.144 MiElHVAC: response data 09000000000000000000000000000000
13:58:29.196 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:29.398 MiElHVAC: response data 0200000003050207000083B446000000
13:58:29.449 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:29.651 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:29.706 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:29.909 MiElHVAC: response data 0200000003050207000083B446000000
13:58:29.912 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:30.115 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:30.166 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:30.368 MiElHVAC: response data 0200000003050207000083B446000000
13:58:30.419 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:30.621 MiElHVAC: response data 09000000000000000000000000000000
13:58:30.675 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:30.878 MiElHVAC: response data 0200000003050207000083B446000000
13:58:30.930 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:31.132 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:31.183 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:31.386 MiElHVAC: response data 0200000003050207000083B446000000
13:58:31.437 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:31.640 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:31.696 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:31.898 MiElHVAC: response data 0200000003050207000083B446000000
13:58:31.949 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:32.151 MiElHVAC: response data 09000000000000000000000000000000
13:58:32.202 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:32.405 MiElHVAC: response data 0200000003050207000083B446000000
13:58:32.456 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:32.659 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:32.666 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:32.869 MiElHVAC: response data 0200000003050207000083B446000000
13:58:32.920 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:33.309 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:33.313 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:33.416 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:33.518 MiElHVAC: response data 0200000003050207000083B446000000
13:58:33.673 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:33.875 MiElHVAC: response data 0200000003050207000083B446000000
13:58:33.927 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:34.128 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:34.180 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:34.383 MiElHVAC: response data 0200000003050207000083B446000000
13:58:34.435 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:34.637 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:34.691 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:34.894 MiElHVAC: response data 0200000003050207000083B446000000
13:58:34.946 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:35.148 MiElHVAC: response data 09000000000000000000000000000000
13:58:35.198 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:35.401 MiElHVAC: response data 0200000003050207000083B446000000
13:58:35.453 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:35.655 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:35.710 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:35.913 MiElHVAC: response data 0200000003050207000083B446000000
13:58:35.916 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:36.118 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:36.171 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:36.373 MiElHVAC: response data 0200000003050207000083B446000000
13:58:36.425 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:36.627 MiElHVAC: response data 09000000000000000000000000000000
13:58:36.683 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:36.886 MiElHVAC: response data 0200000003050207000083B446000000
13:58:36.938 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:37.141 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:37.193 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:37.394 MiElHVAC: response data 0200000003050207000083B446000000
13:58:37.446 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
13:58:37.649 MiElHVAC: response data 030000140001BCBCFE420000787D0000
13:58:37.704 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:37.906 MiElHVAC: response data 0200000003050207000083B446000000
13:58:37.909 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
13:58:38.113 MiElHVAC: response data 09000000000000000000000000000000
13:58:38.163 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:38.366 MiElHVAC: response data 0200000003050207000083B446000000
13:58:38.418 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
13:58:38.619 MiElHVAC: response data 060000000000100EFC00004200000000
13:58:38.674 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
13:58:38.877 MiElHVAC: response data 0200000003050207000083B446000000
13:58:38.928 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a

**Comment by host-felice** [comment]
The AC is turned off from Home Assistant (mqtt climate entity) but tasmota webgui shows it as ON

<img width="356" alt="2023-07-25 (2)" src="https://github.com/arendst/Tasmota/assets/36647269/3875ac70-065e-49bd-8338-489195522652">

**Comment by arendst** [comment]
There you have it:
```
13:57:05.622 SRC: Switch
13:57:05.627 MQT: stat/clima4/RESULT = {"POWER":"ON"}
13:57:05.631 MQT: stat/clima4/POWER = ON
```
It's your stuck switch controlling the power.

**Comment by host-felice** [comment]
Now I went back to 12.4. Same configuration as before. Everything is ok now, no stuck switch.

14:04:46.378 SRC: Switch
14:04:46.384 MQT: stat/clima4/RESULT = {"POWER":"ON"}
14:04:46.388 MQT: stat/clima4/POWER = ON
14:04:46.395 MQT: tele/clima4/HVACSETTINGS = {"Power":"ON","Mode":"cool","HAMode":"cool","Temp":26.0,"FanSpeed":"1","SwingV":"swing","SwingH":"C","Bytes":"0200000103050207000083B446000000"}
14:04:46.428 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
14:04:46.630 MiElHVAC: response data 030000140001BCBCFE420000787D0000
14:04:46.753 CFG: Saved to flash at F8, Count 335, Bytes 4096
14:04:46.756 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
14:04:46.910 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
14:04:47.012 MiElHVAC: response data 0200000103050207000083B446000000
14:04:47.164 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
14:04:47.366 MiElHVAC: response data 0200000103050207000083B446000000
14:04:47.418 MiElHVAC: sending FC42013010 06000000000000000000000000000000 77
14:04:47.620 MiElHVAC: response data 0600000000001A0EFC00004200000000
14:04:47.677 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
14:04:47.879 MiElHVAC: response data 0200000103050207000083B446000000
14:04:47.931 MiElHVAC: sending FC42013010 03000000000000000000000000000000 7a
14:04:48.134 MiElHVAC: response data 030000140001BCBCFE420000787D0000
14:04:48.185 MiElHVAC: sending FC42013010 02000000000000000000000000000000 7b
14:04:48.388 MiElHVAC: response data 0200000103050207000083B446000000
14:04:48.439 MiElHVAC: sending FC42013010 09000000000000000000000000000000 74
14:04:48.643 MiElHVAC: response data 09000000020000000000000000000000

When I turn off the AC, the webconsole shows it as OFF

<img width="320" alt="2023-07-25 (3)" src="https://github.com/arendst/Tasmota/assets/36647269/9eff28e8-3563-4c01-a4be-0849f4511f1f">

**Comment by arendst** [comment, marked as answer]
Ah I see. You need latest dev branch for this. See changelog #18923

**Comment by host-felice** [comment]
@arendst thanks, it is working again with 13.0.3

@Jason2866 as you can see, it was an issue and the retain was and is perfect
